### DISCUSSION TITLE
Problem: After pairing - commands are no longer executed

### DISCUSSION THREAD

**Discussion by comgen89** [Koenkk/zigbee2mqtt#8811] [Devices]
Hello Together,

Last Friday I installed new Paulmann coins. They have connected automatically to zigbee2mqtt. After that I could no longer send commands via Openhab. All devices were therefore without function.

The new devices are listed in the config file, but are not accessible.

**My setting**
- Zibee2mqtt
- mosquitto
- openhab3

**Config.yaml**
_homeassistant: false
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  server: 'mqtt://localhost'
serial:
  port: /dev/ttyACM0
devices:
  '0x00158d000243541a':
    friendly_name: window_contact_05
  '0x00158d00041cbb03':
    friendly_name: window_contact_03
  '0x90fd9ffffec35a74':
    friendly_name: '0x90fd9ffffec35a74'
  '0x00158d000446dfcd':
    friendly_name: window_contact_01
  '0x00158d000447568b':
    friendly_name: motion_sensor_01
  '0x00158d00044c1962':
    friendly_name: window_contact_04
  '0x00158d000446f416':
    friendly_name: window_contact_02
  '0x60a423fffece9623':
    friendly_name: ikea_bulb_01
  '0x842e14fffe0f8ab8':
    friendly_name: ikea_bulb_02
  '0x60a423fffeee9d67':
    friendly_name: ikea_bulb_03
  '0xbc33acfffeae8f09':
    friendly_name: aldi_bulb_filament_01
  '0x680ae2fffe24bfa5':
    friendly_name: ikea_bulb_04
  '0x00158d0006b46d82':
    friendly_name: '0x00158d0006b46d82'
  '0x00158d0006c5e52f':
    friendly_name: '0x00158d0006c5e52f'
  '0x00158d0006c479a9':
    friendly_name: '0x00158d0006c479a9'
  '0x00158d0006c4798e':
    friendly_name: '0x00158d0006c4798e'
  '0x00158d0006c55a8a':
    friendly_name: '0x00158d0006c55a8a'
  '0x00158d0006c680e6':
    friendly_name: '0x00158d0006c680e6'
  '0x00158d0006c5e525':
    friendly_name: '0x00158d0006c5e525'
  '0x00158d0006c67fa5':
    friendly_name: '0x00158d0006c67fa5'
  '0x00158d0006c49786':
    friendly_name: '0x00158d0006c49786'
  '0x00158d0006c5e47c':
    friendly_name: '0x00158d0006c5e47c'_

Can someone help me??

**Comment by zegelin** [comment]
Any messages in the zigbee2mqtt and OpenHAB logs?
Do you have the zigbee2mqtt web frontend UI enabled? If so, can you control the devices via that.

**Reply by comgen89** [reply]
Currently I use MQTT Explorer.

![image](https://user-images.githubusercontent.com/79261043/133969810-81c508a3-8d8f-480b-a82a-4d63b7a5e7e7.png)

The new devices are not even displayed here.

How can I enable web frontend UI?

**Reply by zegelin** [reply]
https://www.zigbee2mqtt.io/information/frontend.html

**Reply by comgen89** [reply]
Thanks!
I have enable the web frontend UI.

Do you know whats _"Interview failed"_.
![image](https://user-images.githubusercontent.com/79261043/133992090-b2aace65-38c7-473f-bc9a-07570edb90ff.png)

**Reply by comgen89** [reply]
If i try to control the ikea_bulb_02. You can see the results below.

**Image:**
![image](https://user-images.githubusercontent.com/79261043/133992497-59a90328-0cce-4cd8-b45d-46291ab118cb.png)


**Text:**
error Publish 'set' 'state' to 'ikea_bulb_02' failed: 'Error: Command 0x842e14fffe0f8ab8/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'MAC no ack' (233))'
info MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'ikea_bulb_02' failed: 'Error: Command 0x842e14fffe0f8ab8/1 genOnOff.on({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (Data request failed with error: 'MAC no ack' (233))'","meta":{"friendly_name":"ikea_bulb_02"},"type":"zigbee_publish_error"}'
